lib/config/config.go

```diff
@@ -215,8 +215,9 @@ type FolderDeviceConfiguration struct {
 
 type OptionsConfiguration struct {
 	ListenAddress           []string `xml:"listenAddress" json:"listenAddress" default:"tcp://0.0.0.0:22000"`
-	GlobalAnnServers        []string `xml:"globalAnnounceServer" json:"globalAnnounceServers" json:"globalAnnounceServer" default:"udp4://announce.syncthing.net:22027, udp6://announce-v6.syncthing.net:22027"`
+	GlobalAnnServers        []string `xml:"globalAnnounceServer" json:"globalAnnounceServers" json:"globalAnnounceServer" default:"https://announce.syncthing.net, https://announce-v6.syncthing.net"`
```

Comment:
Yeah I was considering doing that as well. It could certainly be an option, as if you do enter that as the URL we set `InsecureSkipVerify` and check the ID instead. With the server printing its ID at startup of course. (In this case there is a bona fide certificate, yes.)